tox.ini

```diff
@@ -0,0 +1,41 @@
+[tox]
+minversion = 1.6
+skipdist = True
+envlist = py27,py36,py37,pep8,cover,docs
+
+[testenv]
+setenv = VIRTUAL_ENV={envdir}
+usedevelop = True
+install_command = pip install {opts} {packages}
+
+deps = -r{toxinidir}/requirements.txt
```

Comment:
these requirement files work for python3 as well (as you added the selector to python <= 3.4)